docs/StardustDocs/topics/overview.md

```diff
@@ -15,17 +15,23 @@ The handiness of this abstraction is not in the table itself but in a set of ope
 
 ## Main Features and Concepts
 
-* [**Hierarchical**](hierarchical.md) — `Kotlin Dataframe`  is able to read and present data from different sources including not only plain **CSV** but also **JSON**. That’s why it has been designed hierarchical and allows nesting of columns and cells.
+* [**Hierarchical**](hierarchical.md) — [`DataFrame`](DataFrame.md)  is able to read and present data from different sources including not only plain **CSV** but also **JSON**. That’s why it has been designed hierarchical and allows nesting of columns and cells.
```

Comment:
this is supposed to be the library, not the type, I think

Reply:
Agree